go/pkg/tool/tool_test.go

```diff
@@ -206,3 +208,43 @@ func TestTool_DownloadBlob(t *testing.T) {
 		t.Fatalf("Incorrect content in downloaded file %v, want %v, got %v", fp, want, got)
 	}
 }
+
+func TestTool_UploadBlob(t *testing.T) {
+	e, cleanup := fakes.NewTestEnv(t)
+	defer cleanup()
+	cas := e.Server.CAS
+
+	tmpFile, err := ioutil.TempFile(t.TempDir(), "")
+	if err != nil {
+		t.Fatalf("TempFile failed: %v", err)
+	}
+	if _, err := io.WriteString(tmpFile, "Hello, World!"); err != nil {
```

Comment:
nit: use ioutil.WriteFile instead, it's shorter.

Reply:
done